tests/test_csrf.py

```diff
@@ -0,0 +1,7 @@
+def test_csrf_files(flask_test_client):
+    """
+    The test should return unauthorised message
+     if the request is not received from https server
```

Comment:
Is this true?  I'm not sure this test belongs with other CSRF tests.  What is the purpose of this test?

Reply:
Sure, I will correct it

Reply:
But what is the purpose of the test Ram?

Reply:
I was trying to write a test for unauthorised entry

Reply:
Got rid of this test file for this sprint.  I will try to write a test on later sprints. OWASP zap scan added to .github